connect/governance/governance.py

```diff
@@ -169,9 +169,7 @@ def get_evidence_requirements(self, tags: dict = None, verbose=False):
         if tags is None:
             tags = self.get_model_tags()
 
-        reqs = [
-            e for e in self._evidence_requirements if (not e.tags or e.tags == tags)
-        ]
+        reqs = [e for e in self._evidence_requirements if check_subset(e.tags, tags)]
```

Comment:
Can you update the docstring of `get_evidence_requirements` to reflect the latest change. I would also add the example you put in the PR.